chat-plugins/uno.js

```diff
@@ -452,6 +452,9 @@ exports.commands = {
 			if (!this.can("minigame", null, room)) return;
 			if (room.unoDisabled) return this.errorReply("UNO is disabled for this room.");
 			if (room.game) return this.errorReply("There is already a game in progress in this room.");
+			let num = (target === '') ? 3 : parseInt(target);
```

Comment:
why are the parens around ``target === ''``??  if you're using parens at all in this line (which you really don't need to), then put them around the entire ternary operation